internal/ethapi/api.go

```diff
@@ -1240,6 +1240,8 @@ func RPCMarshalBlock(block *types.Block, inclTx bool, fullTx bool, config *param
 			}
 		}
 		fields["transactions"] = transactions
+		// inclTx also expands withdrawals
```

Comment:
Might be misremembering, but I think a while back we discussed extending the RPC parameter signature to have another boolean for including withdrawals. This is okay for now, but we should keep that in mind.